core-bundle/src/Command/ResizeImagesCommand.php

```diff
@@ -118,7 +118,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         if (!$output instanceof ConsoleOutputInterface) {
-            throw new \InvalidArgumentException(sprintf('Output must be an instance of "%s"', ConsoleOutputInterface::class));
+            throw new InvalidArgumentException(sprintf('Output must be an instance of "%s"', ConsoleOutputInterface::class));
```

Comment:
Since `$output` is not a command line argument provided by the user, `\InvalidArgumentException` seems correct to me here.

Reply:
:+1: Agreed. 

Adjusted in cfc71897b43a7ddb5f79a7a9ee16435d81bab936.